source/intel.js

```diff
@@ -532,9 +528,17 @@ function NeptunesPrideAgent() {
 		let now = new Date();
 		let arrival = new Date(now.getTime() + msplus);
 		let p = prefix !== undefined ? prefix : "ETA ";
-		let ttt = p + ampm(arrival.getHours(), arrival.getMinutes());
-		if (arrival.getDay() != now.getDay())
-			ttt = p + days[arrival.getDay()] + " @ " + ampm(arrival.getHours(), arrival.getMinutes());
+		let ttt = '';
+		let totalETA;
+		if (!NeptunesPride.gameConfig.turnBased) {
+			ttt = p + ampm(arrival.getHours(), arrival.getMinutes());
+			if (arrival.getDay() != now.getDay())
+				ttt = p + days[arrival.getDay()] + " @ " + ampm(arrival.getHours(), arrival.getMinutes());
+		}
+		else {
+			totalETA = (arrival - now);
+			ttt = p + Crux.formatTime(totalETA);
```

Comment:
Got it; this feature only effects turn based games.

Reply:
yep! I think if we'd like to add any further time formats we'd need to convert the % hot key to cycle through rather than toggle

Reply:
Are these changes still only in-effect when the hotkey '%' is pressed?

Reply:
this particular change - yes. 
1) for real time games it goes from days/hours/minutes/s to ETA time (this logic was unchanged)
2) for turn based, it goes from days/hours to days/hours + " - x turns" since an ETA time is not very useful
    a) that being said, its possible it *could* be useful to have the ETA time be based on time until next turn or something, 
     but I think it would not add value for turn based